drivers/overlay/ostweaks_linux.go

```diff
@@ -21,10 +45,28 @@ func writeSystemProperty(key, value string) error {
 	return ioutil.WriteFile(path.Join("/proc/sys", keyPath), []byte(value), 0644)
 }
 
+func readSystemProperty(key string) ([]byte, error) {
+	keyPath := strings.Replace(key, ".", "/", -1)
+	return ioutil.ReadFile(path.Join("/proc/sys", keyPath))
+}
+
 func applyOStweaks() {
-	for k, v := range sysctlConf {
-		if err := writeSystemProperty(k, v); err != nil {
-			logrus.Errorf("error setting the kernel parameter %s = %s, err: %s", k, v, err)
+	for k, v := range osConfig {
+		// read the existing property from disk
+		oldv, err := readSystemProperty(k)
+
+		if err != nil {
+			logrus.Errorf("error reading the kernel parameter %s = %s, err: %s", k, v, err)
```

Comment:
maybe you can remove the value here, that is the new one, does not have any correlation with the read

Reply:
Oh good catch. Didn't consider that.